modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -6861,38 +7108,922 @@ public void testPutCustomObjectEntryWithNestedCustomObjectEntriesInOneToManyRela
+		);
+
+		HTTPTestUtil.customize(
+		).withCredentials(
+			"test1@liferay.com", "test1"
+		).apply(
+			() ->
+				oneToManyJSONObjects[3] = HTTPTestUtil.invokeToJSONObject(
+					JSONUtil.put(
+						"externalReferenceCode", "ERC1"
+					).toString(),
+					endpoint2, Http.Method.POST)
+		);
+
+		JSONObject jsonObject1 = HTTPTestUtil.invokeToJSONObject(
+			JSONFactoryUtil.getNullJSON(
+			).toString(),
+			endpoint1, Http.Method.POST);
+
+		JSONObject jsonObject2 = HTTPTestUtil.invokeToJSONObject(
+			JSONFactoryUtil.getNullJSON(
+			).toString(),
+			endpoint1, Http.Method.POST);
+
+		HTTPTestUtil.invokeToJSONObject(
+			null,
+			String.format(
+				"%s/%d/%s/%d", endpoint1, jsonObject1.getLong("id"),
+				_objectRelationship1.getName(),
+				oneToManyJSONObjects[0].getLong("id")),
+			Http.Method.PUT);
+
+		HTTPTestUtil.invokeToJSONObject(
+			null,
+			String.format(
+				"%s/%d/%s/%d", endpoint1, jsonObject1.getLong("id"),
+				_objectRelationship1.getName(),
+				oneToManyJSONObjects[1].getLong("id")),
+			Http.Method.PUT);
+
+		HTTPTestUtil.invokeToJSONObject(
+			null,
+			String.format(
+				"%s/%d/%s/%d", endpoint1, jsonObject2.getLong("id"),
+				_objectRelationship1.getName(),
+				oneToManyJSONObjects[2].getLong("id")),
+			Http.Method.PUT);
+
+		HTTPTestUtil.invokeToJSONObject(
+			null,
+			String.format(
+				"%s/%d/%s/%d", endpoint1, jsonObject2.getLong("id"),
+				_objectRelationship1.getName(),
+				oneToManyJSONObjects[3].getLong("id")),
+			Http.Method.PUT);
+
+		HTTPTestUtil.customize(
+		).withCredentials(
+			"test3@liferay.com", "test3"
+		).apply(
+			() ->
+				oneToManyJSONObjects[0] = HTTPTestUtil.invokeToJSONObject(
+					JSONUtil.put(
+						_OBJECT_FIELD_NAME_2, RandomTestUtil.randomString()
+					).toString(),
+					endpoint2 + "/by-external-reference-code/ERC4",
+					Http.Method.PATCH)
+		);
+
+		HTTPTestUtil.customize(
+		).withCredentials(
+			"test2@liferay.com", "test2"
+		).apply(
+			() ->
+				oneToManyJSONObjects[1] = HTTPTestUtil.invokeToJSONObject(
+					JSONUtil.put(
+						_OBJECT_FIELD_NAME_2, RandomTestUtil.randomString()
+					).toString(),
+					endpoint2 + "/by-external-reference-code/ERC2",
+					Http.Method.PATCH)
+		);
+
+		try {
+			_testSortByFieldName(
+				endpoint1, jsonObject2, jsonObject1,
+				String.format("%s/creator", _objectRelationship1.getName()));
+			_testSortByFieldName(
+				endpoint1, jsonObject1, jsonObject2,
+				String.format(
+					"%s/dateCreated", _objectRelationship1.getName()));
+			_testSortByFieldName(
+				endpoint1, jsonObject2, jsonObject1,
+				String.format(
+					"%s/dateModified", _objectRelationship1.getName()));
+			_testSortByFieldName(
+				endpoint1, jsonObject2, jsonObject1,
+				String.format(
+					"%s/externalReferenceCode",
+					_objectRelationship1.getName()));
+			_testSortByFieldName(
+				endpoint1, jsonObject1, jsonObject2,
+				String.format("%s/id", _objectRelationship1.getName()));
+
+			// TODO Uncomment when LPD-20288 is fixed
```

Comment:
Expect the failures
```AssertUtils.assertFailure(Exception.class, "kdlf sdkf l", () -> 
			_testSortByFieldName(
				endpoint1, jsonObject2, jsonObject1,
				String.format("%s/creatorId", _objectRelationship1.getName())));```
This will fail when the issue is fixed so no neee to remember.

Reply:
Done. I had to create a new method as the already existing method in the Utils does not catch a Throwable, it catches an exception, and I don't want to update the test